lib/trento/discovery/policies/cluster_policy.ex

```diff
@@ -157,11 +160,13 @@ defmodule Trento.Discovery.Policies.ClusterPolicy do
            sbd: sbd,
            cluster_type: ClusterType.hana_scale_out(),
            hana_architecture_type: HanaArchitectureType.classic(),
+           additional_sids: additional_sids,
            sid: sid
          } = payload
        ) do
     %{
       architecture_type: HanaArchitectureType.classic(),
+      hana_scenario: parse_hana_scenario(additional_sids),
```

Comment:
I think you could directly put `HanaScenario.performance_optimized()` here, as we don't have any scale out cost optimized.
I guess using the function will get us to the same result.
Up to you